tests/test_encoder.py

```diff
@@ -258,3 +259,29 @@ def test_dump_to_file(tmpdir):
         dump([1, 10], fp)
 
     assert path.read_binary() == b'\x82\x01\x0a'
+
+
+@pytest.mark.parametrize('value, expected', [
+    (OrderedDict([(b'a', b''), (b'b', b'')]), 'A2416140416240'),
+    (OrderedDict([(b'b', b''), (b'a', b'')]), 'A2416140416240'),
+    (OrderedDict([(u'a', u''), (u'b', u'')]), 'a2616160616260'),
+    (OrderedDict([(u'b', u''), (u'a', u'')]), 'a2616160616260'),
+    (OrderedDict([(b'00001', u''), (b'002', u'')]), 'A2433030326045303030303160'),
+    (OrderedDict([(255, 0), (2, 0)]), 'a2020018ff00')
+], ids=[
```

Comment:
Put the ids on the same line or wrap on the following lines only when they go over the 99 column limit.